.github/workflows/npm-app-release.yml

```diff
@@ -252,6 +252,19 @@ jobs:
           git config --global --add safe.directory /__w/$dname/$dname
         shell: sh
 
+      - name: Download artifact
+        uses: actions/download-artifact@v3
+        with:
+          name: ${{ needs.build-artifact.outputs.artifact }}
+          path: built-artifacts/
+        continue-on-error: false
```

Comment:
Agree this is a good change 👍 Shall we copy this change to `npm-lib-release` and any others before merging?

Reply:
yes, I will